test/Microsoft.SqlTools.ServiceLayer.TestDriver/Tests/LanguageServiceTests.cs

```diff
@@ -279,5 +279,137 @@ public class LanguageServiceTests : TestBase
                 WaitForExit();
             }
         }
+
+        [Fact]
+        public async Task FunctionSignatureCompletionReturnsCorrectFunction()
+        {
+            string sqlText = "EXEC sys.fn_isrolemember ";
+            string ownerUri = System.IO.Path.GetTempFileName();
+
+            try
+            {
+                System.IO.File.WriteAllText(ownerUri, sqlText);
```

Comment:
>System.IO. [](start = 16, length = 10)

nitpick: use System.IO and remove the fully-qualified name unless there's a conflict?